client/app/components/Header/__tests__/Header.spec.jsx

```diff
@@ -13,22 +14,30 @@ const getComponent = () => (
   />
 );
 
-const wrapper = mount(getComponent());
-
 describe('Header', () => {
   it('renders correctly', () => {
-    expect(wrapper.length).toEqual(1);
+    render(component);
+
+    expect(screen.getByRole('banner')).toBeInTheDocument();
+    expect(screen.getByRole('navigation')).toBeInTheDocument();
+    expect(screen.getByRole('link', { name: /home/i })).toBeInTheDocument();
+    expect(screen.getByRole('link', { name: /link 1/i })).toBeInTheDocument();
+    expect(screen.getByRole('link', { name: /link 1/i })).toHaveClass(
+      'headerActiveLink',
+    );
+    expect(screen.getByRole('link', { name: /link 2/i })).toBeInTheDocument();
   });
 
   it('toggles hamburger correctly', () => {
-    wrapper.find('#headerHamburger').simulate('click');
-    expect(wrapper.find('#headerMobile').length).toEqual(1);
-    wrapper.find('#headerHamburger').simulate('click');
-    expect(wrapper.find('#headerMobile').length).toEqual(0);
-  });
+    const { container } = render(component);
+    const hamburger = container.querySelector('#headerHamburger');
+    expect(hamburger).toBeInTheDocument();
```

Comment:
I think we can safely remove this assertion since the rest of the assertions should be enough.